tests/unit/amo/components/TestRatingsByStar.js

```diff
@@ -218,25 +246,38 @@ describe(__filename, () => {
       addon,
       location: createFakeLocation({ query: { src } }),
     });
-    const counts = root.find('.RatingsByStar-count').find(Link);
+    const tableRow = root.find('.RatingByStar-table-row');
 
-    function validateLink(link, score) {
-      expect(link).toHaveProp(
-        'to',
-        reviewListURL({ addonSlug: addon.slug, score, src }),
+    const clientApp = CLIENT_APP_ANDROID;
```

Comment:
See my comment above about setting these in the test.

Reply:
Yes! I updated this too.